hardhat.config.ts

```diff
@@ -1,7 +1,51 @@
 import { task } from "hardhat/config";
 import "@nomiclabs/hardhat-waffle";
 import "@nomiclabs/hardhat-ethers";
-import "solidity-coverage"
+import "solidity-coverage";
+
+const lazyImport = async (module: string) => {
+	const importedDefault = await import(module);
+	return importedDefault.default;
+}
+
+task("deploy-registry", "Deploys a Registry contract")
+	.setAction(async taskArgs => {
+		const registryDeployer = await lazyImport('./scripts/registry');
+		
+		await registryDeployer();
+	});
+
+task("deploy-factory", "Deploys a Refunder's Factory")
+	.addParam("registryAddress", "The address of refunders Registry")
```

Comment:
lets make the param `registry` only and not `registryAddress`. Same thing for `factoryAddress` and `masterRefunderAddress`. 
It is redundant to have `address` in the property imo